### PR TITLE
Softened the verification setup colors

### PR DESCRIPTION
**Pull request by drfarrell**
### Description

Softened the colors to be a bit less jarring when the project is setup

### What is the purpose of this pull request? 

- [ ] New feature
- [ ] Documentation update
- [X] Bug fix
- [ ] Refactor
- [ ] Release
- [ ] Other

### CODE REVIEW

File: app/src/components/Context/index.tsx [hunk 1]
```diff
@@ -2,19 +2,23 @@ import { AuthManager } from '@/lib/auth';
...
 const RouteContext = createContext(routeManager);
 const ProjectsContext = createContext(projectsManager);
 const EditorEngineContext = createContext(editorEngine);
+const UpdateContext = createContext(updateManager);
 
 export const useAuthManager = () => useContext(AuthContext);
 export const useRouteManager = () => useContext(RouteContext);
 export const useProjectsManager = () => useContext(ProjectsContext);
 export const useEditorEngine = () => useContext(EditorEngineContext);
+export const useUpdateManager = () => useContext(UpdateContext);
```
Comment (Kitenite) [inline app/src/components/Context/index.tsx:24]: Wtf, was this always missing?

Comment (drfarrell) [reply app/src/components/Context/index.tsx:24]: Seems like it! I tossed it in because I couldn't get the update context without it.

Comment (Kitenite) [reply app/src/components/Context/index.tsx:24]: Ah I see...

Comment (Kitenite) [reply app/src/components/Context/index.tsx:24]: It is duplicated. I like your pattern better so I'll remove the existing one.

**Review by Kitenite — APPROVED** [review]
Looks good to merge besides the question mark feeling like a tooltip.

**Pull request merged**